src/widgetastic_patternfly4/descriptionlist.py

```diff
@@ -0,0 +1,32 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Widget
+
+
+class BaseDescriptionList:
+    """Represents the Patternfly Description list.
+
+    https://www.patternfly.org/v4/components/description-list
+    """
+
+    TERM_LOCATOR = ".//dt"
+    DESC_LOCATOR = ".//dd"
+
+    def read(self):
+        """Return a dictionary of term / description pairs."""
+        terms = [
+            self.browser.text(el) for el in self.browser.elements(self.TERM_LOCATOR, parent=self)
```

Comment:
try to avoid `self` as a parent. it will auto resolved this.

Reply:
fixed, thanks!